### PR TITLE
Coe 924 oee simulators prepare new profile for export import test

### PR DESCRIPTION
**Pull request by namnhatpham**

### CODE REVIEW

File: simulators/README.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Note: A profile will be created and activated only if no other profiles are alre
 
 ### Deployment
 
-To deploy this project, upload the zip file to the Cumulocity as Microservice. The zip file can be created locally as described above or downloaded from the [releases](releases) section.
+To deploy this project, upload the zip file to the Cumulocity as Microservice. The zip file can be created locally as described above or downloaded from the [releases](https://github.com/SoftwareAG/oee-simulators/releases) section.
```
Comment (mvoigt-sag) [inline simulators/README.md:104]: why is an absolute URL required here?

File: simulators/extras/ExportProfileData.py [hunk 2]
```diff
@@ -116,6 +119,9 @@ def ExportMeasurements(deviceId, createFrom, createTo, filePath):
 
 def ListMeasurements(deviceId, createFrom, createTo):
     jsonMeasurementsList = []
+    if TEST_FLAG:
+        createFrom = None
+        createTo = None
```
Comment (mvoigt-sag) [inline simulators/extras/ExportProfileData.py:124]: why is this needed here?

**Review by mvoigt-sag — CHANGES_REQUESTED** [review]
- it seems there is still one test failing: https://github.com/SoftwareAG/oee-simulators/actions/runs/4701483952/jobs/8337554829?pr=85
- if we want to get the checks to pass & merge this PR, we could disable the failing tests for now and fix them in another PR

File: simulators/README.md (hunk 1 above)
Comment (namnhatpham) [reply simulators/README.md:104]: Because there is no release directory in the project so the link can not be redirected and will show issues when users read documents and click. Add absolute URL here will connect user to the releases page on github and they can download them during the study of document.

**Review by mvoigt-sag — APPROVED** [review]
